ipa-core/src/helpers/transport/query/mod.rs

```diff
@@ -260,43 +259,45 @@ impl FromStr for DPParams {
     }
 }
 
-// TODO: naming: s/TestingWithNoDp/no-dp
-impl Display for DPParams {
+impl Display for DpParams {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            DPParams::TestingWithNoDP => write!(f, "TestingWithNoDP"),
-            DPParams::WithDP(eps) => write!(f, "WithDP={eps}"),
+            DpParams::NoDp => write!(f, "NoDp"),
+            DpParams::WithDp(eps) => write!(f, "WithDp={eps}"),
         }
     }
 }
 
-impl Serialize for DPParams {
-    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error> where S: Serializer {
+impl Serialize for DpParams {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
         serializer.serialize_str(&self.to_string())
     }
 }
 
-impl<'de> Deserialize<'de> for DPParams {
-    fn deserialize<D: Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error>
-    {
+impl<'de> Deserialize<'de> for DpParams {
+    fn deserialize<D: Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error> {
         let s: &str = Deserialize::deserialize(deserializer)?;
-        Self::from_str(s).map_err(|e| de::Error::custom(format!("failed to deserialize DpParams object: {e}")))
+        Self::from_str(s)
+            .map_err(|e| de::Error::custom(format!("failed to deserialize DpParams object: {e}")))
     }
 }
 
 #[cfg(test)]
-impl PartialEq<Self> for DPParams {
-    fn eq(&self, other: &DPParams) -> bool {
+impl PartialEq<Self> for DpParams {
```

Comment:
Isn't this something you can use `#[derive(PartialEq)]` for?

Reply:
I can go back and see if that will work again.  Some merge conflict or something breaking things online I need to fix first.

Reply:
yeah, looks like it works with this derived.